mlir/lib/Transforms/SCFVectorize.cpp

```diff
@@ -0,0 +1,648 @@
+//===- SCFVectorize.cpp - SCF vectorization utilities ---------------------===//
```

Comment:
This should go to `mlir/lib/Dialect/SCF/Transforms`

Reply:
done